cpp/examples/sde_seir2v.cpp

```diff
@@ -0,0 +1,96 @@
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV1V2)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV1V2)}];
+
+    double beta = mio::DistributionAdapter<std::lognormal_distribution<double>>::get_instance()(rng, std::log(0.08), 0.5);
+    double kappa = mio::DistributionAdapter<std::lognormal_distribution<double>>::get_instance()(rng, std::log(8), 0.2);
+    double gamma = mio::DistributionAdapter<std::lognormal_distribution<double>>::get_instance()(rng, std::log(16), 0.2);
+
+    model.parameters.set<mio::sseir2v::TransmissionProbabilityOnContactV1>(beta);
+    model.parameters.set<mio::sseir2v::TransmissionProbabilityOnContactV2>(beta);
+    model.parameters.set<mio::sseir2v::TimeExposedV1>(kappa);
+    model.parameters.set<mio::sseir2v::TimeExposedV2>(kappa);      
+    model.parameters.set<mio::sseir2v::TimeInfectedV1>(gamma);
+    model.parameters.set<mio::sseir2v::TimeInfectedV2>(gamma);
+
+    model.check_constraints();
+
+    auto ssirs = mio::sseir2v::simulate(t0, 23, dt, model);
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::Susceptible)}] = ssirs.get_value(static_cast<size_t>(ssirs.get_num_time_points()) - 1)[0];
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV1)}] = ssirs.get_value(static_cast<size_t>(ssirs.get_num_time_points()) - 1)[1];
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV1)}]  = ssirs.get_value(static_cast<size_t>(ssirs.get_num_time_points()) - 1)[2];
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV1)}] = ssirs.get_value(static_cast<size_t>(ssirs.get_num_time_points()) - 1)[3];    
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV2)}]  = ssirs.get_value(static_cast<size_t>(ssirs.get_num_time_points()) - 1)[4];
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV2)}]  = 10;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV2)}] = ssirs.get_value(static_cast<size_t>(ssirs.get_num_time_points()) - 1)[6];
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV1V2)}]  = ssirs.get_value(static_cast<size_t>(ssirs.get_num_time_points()) - 1)[7];
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV1V2)}]  = ssirs.get_value(static_cast<size_t>(ssirs.get_num_time_points()) - 1)[8];
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV1V2)}] = ssirs.get_value(static_cast<size_t>(ssirs.get_num_time_points()) - 1)[9];
+    auto ssirs2 = mio::sseir2v::simulate(tmid, tmax, dt, model);
```

Comment:
The "23" in the first simulate was meant to be a tmid, right?

```suggestion
    auto ssirs                                                    = mio::sseir2v::simulate(t0, 23, dt, model);
    model.populations.array()                                     = ssirs.get_last_value().cast<mio::UncertainValue>();
    model.populations[{mio::sseir2v::InfectionState::InfectedV2}] = 10;
    auto ssirs2                                                   = mio::sseir2v::simulate(tmid, tmax, dt, model);
```